support-frontend/assets/helpers/forms/paymentIntegrations/readerRevenueApis.js

```diff
@@ -339,31 +330,8 @@ function postRegularPaymentRequest(
     });
 }
 
-function setPasswordGuest(
-  password: string,
-  guestAccountRegistrationToken: string,
-  csrf: CsrfState,
-): Promise<boolean> {
-
-  const data = { password, guestAccountRegistrationToken };
-  return logPromise(fetch(`${routes.contributionsSetPasswordGuest}`, requestOptions(data, 'same-origin', 'PUT', csrf)))
```

Comment:
if we no longer use the endpoint, will we get rid of it completely?

Reply:
good spot, will do